package/kedro_viz/api/rest/requests.py

```diff
@@ -7,5 +7,6 @@ class DeployerConfiguration(BaseModel):
     """Credentials for Deployers."""
 
     platform: str
+    is_all_previews_enabled: bool = False
```

Comment:
![image](https://github.com/kedro-org/kedro-viz/assets/87735534/f264c357-4748-43ef-bab8-48a75f0a6557)

These are not related to your change, but the file permissions are somehow getting changed. Can you do -

`chmod 644 package/kedro_viz/api/rest/requests.py` and push the changes ? @jitu5

Reply:
done.